src/utils/asset.ts

```diff
@@ -4,10 +4,10 @@ import { CHAIN_ID } from './chain';
 import { PLR_ADDRESS_PER_CHAIN } from '../constants/assetConstants';
 
 export const plrDaoAsset: IAssetWithBalance = {
-  address: '0xa6b37fc85d870711c56fbcb8afe2f8db049ae774',
+  address: '0x6D5862a18C6a169D44d02a8B726a02A5B707B484',
```

Comment:
as per comment in the above you can then leave `plrDaoAsset` as is and just have DKU mapped from separate constant var

Reply:
sorry @parag-chauhan, but these changes seems to be just same `plrDaoAsset`, but with demo token values?

Reply:
my suggestion was to have separate and leave actual token constants as the same as it might be used in other pieces of code so be careful